jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/BottomSheet.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.BottomSheetAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+/**
+ * To see an example of BottomSheet web element please visit https://material.angular.io/components/bottom-sheet/overview.
+ */
+
+public class BottomSheet extends UIBaseElement<BottomSheetAssert> {
+    public String BOTTOM_SHEET_VALUES_CSS_SELECTOR = "#bottom-sheet-container .mat-list-text span:first-child";
```

Comment:
Code convention not respected. The field name must be written in lowercase camelcase.